features/scripts/readme.md

```diff
@@ -0,0 +1,19 @@
+## Scripts
+
+Scripts used for setting up tests
+
+### Removing the Temporary Access User
```

Comment:
Do we need to say this? It might be confusing for users when we go to major version `1`. I feel like it's not necessary to mention previous behaviors.